src/Tizen.WebView/Tizen.WebView/Settings.cs

```diff
@@ -98,5 +115,76 @@ public int DefaultFontSize
                 Interop.ChromiumEwk.ewk_settings_default_font_size_set(_handle, value);
             }
         }
+ 
+        /// <summary>
+        /// Whether the script can open windows.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool ScriptsCanOpenWindows
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_scripts_can_open_windows_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_scripts_can_open_windows_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Whether the viewport meta element is enabled.
+        /// If true, Viewport meta element is applied.
+        /// If false, Viewport meta element is ignored.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool ViewportEnabled
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_viewport_meta_tag_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_viewport_meta_tag_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Whether the autofill enabled.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+	[Flags]
+	public enum AutofillModes
```

Comment:
do not use inner enum

please extract to outside of this class

Reply:
done